shared-lib/spiutils/src/cmds.rs

```diff
@@ -0,0 +1,86 @@
+/// SPI flash op codes
+#[repr(u8)]
+#[allow(dead_code)]
+pub enum OpCodes {
+    /// No operation
+    Nop = 0x00,
+
+
```

Comment:
Too many blank lines here? Probably should just be 1.

Reply:
I was using that to visually distinguish commands from each other. I added some additional comment in.